src/pages/block/[block_height].tsx

```diff
@@ -704,3 +706,13 @@ export default function Block() {
     </Layout>
   );
 }
+
+export async function getServerSideProps() {
+  const envs = getClientSideEnvs();
+
+  return {
+    props: {
+      envs,
+    },
+  };
+}
```

Comment:
Two thoughts:

- Can we put this at a kind of parent layout component so we don't have to do this for every page we create (and need to remember or else risk errors)?
- What if we created an backend API route for returning this and we put it in react-query (client side) instead?